### PR TITLE
Add route to connect to database

### PR DESCRIPTION
**Pull request by alexzhaohong**
I added routes! Pretty simple.

Next up: Integrate Jest and enzyme

### CODE REVIEW

**Comment by alexzhaohong** [comment]
Screen Shot 2020-05-01 at 11 04 11 PM
This is what the client looks like right now!

File: client/dist/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Front-End Capstone Reviews Module</title>
+  <link rel="icon" href="/favicon.ico" type="image/x-icon" />
+  <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
+</head>
+
+<body>
+  <img src="Space-Lobster.jpg" alt="Group 9 Space Lobster Mascot" title="Space Lobster">
```
Comment (robertheler) [inline client/dist/index.html:14]: love the lobster client

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -1,6 +1,33 @@
...
+    let url = `/api/models/${this.state.productcode}/reviews`;
+    fetch(url)
+      .then(response => response.json())
+      .then(reviewsData => this.changeReviews(reviewsData))
+      .catch(err => console.error(err));
+  }
+
+  // Set State
+  changeReviews(reviews) {
+    this.setState({ reviews: reviews });
```
Comment (robertheler) [inline client/src/components/App.jsx:28]: In the sprit of ES6, I wonder if you can just say  `this.setState({reviews});`. I'm actually not sure if you can do that, let me know

File: server/index.js [hunk 3]
```diff
@@ -1,12 +1,29 @@
...
+// app.use(morgan('dev'));
+app.use(express.json());
+app.use(middleware.httpRequestLogger);
+
+app.get('/api', (req, res) => {
+  console.log('hello inside get route');
+  res.send('response sent correctly!');
+});
+
+app.use('/api/models', router);
```
Comment (robertheler) [inline server/index.js:24]: I don't know if `models` should be in the url. From the perspective of an API user who wants to access the reviews, typing in models doesn't make sense.

**Pull request merged**